tests/test_plottingfuncs.py

```diff
@@ -196,3 +196,21 @@ def test_plot_and_save_non_square_bounding_box(
         region_properties=minicircle_grain_region_properties_post_removal,
     ).plot_and_save()
     return fig
+
+
+RNG = np.random.default_rng(seed=1000)
+array = RNG.random((10, 10))
+mask = RNG.uniform(low=0, high=1, size=array.shape) > 0.5
+
+@pytest.mark.mpl_image_compare(baseline_dir="resources/img/")
+def test_mask_cmap(plotting_config: dict, tmp_path: Path) -> None:
+    """Test the plotting of a mask with a different colourmap (blu)."""
+    plotting_config['mask_cmap'] = 'blu'
+    fig, _ = Images(
+        data=array,
+        output_dir=tmp_path,
+        filename="colour.png",
+        data2=mask,
```

Comment:
Test is failing because `data2` which wasn't a very informative option name was changed to `masked_array`. Wouldn't have shown as a merge conflict as this is a new test.

```suggestion
        masked_array=mask,
```

Reply:
I've changed this because it now doesn't have to be a binary masked array as the colourmap can take maps and not just single colours, and I thought that it might confuse people. 

I see now that I've kind of confused myself so will change all back to masked array (as technically the array can be anything)